lib/rubocop/cop/style/struct_inheritance.rb

```diff
@@ -33,6 +35,32 @@ def on_class(node)
           {(send (const nil? :Struct) :new ...)
            (block (send (const nil? :Struct) :new ...) ...)}
         PATTERN
+
+        def autocorrect(node)
```

Comment:
I find it helpful to use a more specific name, in this case maybe `class_node`. That helps me remember what the autocorrect method is working on without looking up the `add_offense` calls.

Reply:
That's a great tip, thanks. I've updated the code.